content/en/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions.md

```diff
@@ -418,10 +427,12 @@ The following types are available to be used in the `type` field of columns (com
 - `date` – renderend differentially as time since this timestamp.
 
 If the value inside a CustomResource does not match the type specified for the column,
-the value is omitted. Hence, it is highly suggested to use CRD validation to ensure
-that the value types are correct.
+the value is omitted. Use CustomResourceDefinition validation to ensure that the value
```

Comment:
I think calling it CustomResource validation will be better. Wdyt?

Reply:
+1

Reply:
fixed